client/src/components/SignUp/login.jsx

```diff
@@ -42,6 +42,9 @@ class Login extends React.Component {
   // }
   submitForm(e) {
     e.preventDefault();
+    if(this.state.email === '' || this.state.password === '') {
```

Comment:
could probably be shortened to if (!this.state.email || !this.state.password) {...}